haas/cli.py

```diff
@@ -175,16 +205,16 @@ def list_projects():
     do_get(url)
 
 @cmd
-def project_add_user(project, user):
+def user_add_project(user, project):
```

Comment:
Could we stick with project\_*\_user?

We have many other project_... calls, so keeping this project_ would seem more consistent.